opent2t-reference/main.js

```diff
@@ -0,0 +1,283 @@
+var electron = require('electron');
+var app = electron.app;
+var BrowserWindow = electron.BrowserWindow;
+var q = require('q');
+var fs = require('fs');
+require('electron-reload')(__dirname);
+var Opent2tHelper = require("./Opent2tHelper");
+var opent2tHelper = new Opent2tHelper();
+var helpers = require('./helpers');
+
+
+// Keep a global reference of the window object, if you don't, the window will
+// be closed automatically when the JavaScript object is GCed.
+var mainWindow = null;
+
+// Quit when all windows are closed.
+app.on('window-all-closed', function () {
+    // On OS X it is common for applications and their menu bar
+    // to stay active until the user quits explicitly with Cmd + Q
+    if (process.platform != 'darwin') {
+        app.quit();
+    }
+});
+
+app.on('ready', function () {
+    // Create the browser window.
+    mainWindow = new BrowserWindow({ width: 1198, height: 750, frame: true, show: false, autoHideMenuBar: true });
+    //mainWindow.$ = mainWindow.jQuery = require('./lib/jquery-2.1.4.min.js');
+
+    mainWindow.once('ready-to-show', () => {
+        mainWindow.show();
+    });
+
+    // and load the index.html of the app.
+    mainWindow.loadURL('file://' + __dirname + '/index.html');
+
+    // Open the devtools.
+    //mainWindow.openDevTools();
```

Comment:
Removed commented-out code.

---
In reply to: [101374869](https://github.com/openT2T/opent2t-cli/pull/16#discussion_r101374869) [](ancestors = 101374869)